<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCAShapeModelEstimatorView</class>
 <widget class="QWidget" name="PCAShapeModelEstimatorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="DirecotyLE"/>
      </item>
      <item>
       <widget class="QPushButton" name="BrowseBtn">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Output Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="OutDirLE"/>
      </item>
      <item>
       <widget class="QPushButton" name="OutDirBrowseBtn">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="ImageList">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ApplyBtn">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
